readme.md

```diff
@@ -1,30 +1,261 @@
-# Usage
+# @bitnoi.se/scheduler
 
-In order to display Scheduler properly it's required to import styles:
+Typescript oriented, light-weight and ultra fast React Component for creating a gantt charts.
+
+### Installation
+
+```bash
+# yarn
+yarn add @bitnoi.se/scheduler
+# npm
+npm install @bitnoi.se/scheduler
+```
+
+### Example Usage
+
+1. import required styles for scheduler
+
+```ts
+import "@bitnoi.se/react-scheduler/dist/style.css";
+```
+
+2. Import Scheduler component into your project
 
 ```ts
-import "@bitnoi.se/react-scheduler/style.css";
+import { Scheduler } from "@bitnoi.se/scheduler"
+import type { SchedulerData } from "@bitnoi.se/scheduler/types/global"
+
+default export function Component() {
+	return (
```

Comment:
Please adjust indents here as in `mockedSchedulerData` - 2 spaces should be perfect